### PR TITLE
ACCUMULO-4313 - Made several improvements to website:

### PR DESCRIPTION
**Pull request by mikewalch**
* Put website in container
 * Moved Accumulo logo to navbar
 * Organized navbar
 * Moved all ASF links to their own dropdown on right
 * Added logo to home page and simplified text
 * Reduced text in footer
 * Put new window icons after links
 * Renamed 'GitHub Issues' link in navbar to 'Pull Requests'
 * Pages have a default font size of 14px rather 16px
 * Sidebar with social links only visible on home page so
   documentation fills page which improves readability
 * Removed download link from sidebar and added to navbar

### CODE REVIEW

File: _includes/footer.md [hunk 1]
```diff
@@ -4,10 +4,7 @@
 [![Apache Software Foundation][ASF_IMG]{: #asf-logo height="100"}][ASF]
 
 Copyright &copy; 2011-{{ site.time | date: '%Y' }} The Apache Software
-Foundation, Licensed under the [Apache License, Version 2.0][AL2]. Apache
```
Comment (joshelser) [inline _includes/footer.md]: Did you move this block about trademarks somewhere? It looks like it was deleted -- we definitely do not want to do that.

Comment (mikewalch) [inline _includes/footer.md]: I was trying to reduce the legalese in the footer as its on every page.  However, I see your point that it should be on the website somewhere.  I pushed another commit ( ff8f675615 ) where I created a notice page.  This is now linked to in the footer.  Any additional legal notices can be added here rather than putting them in the footer.  Does that work?

Comment (joshelser) [inline _includes/footer.md]: I think we should still have the legalese on the index page as it is expected to be present. We probalby don't need it in every footer. Maybe we can include that "partial" (I forget what Jekyll calls it) in the footer just on the index page?

Comment (mikewalch) [inline _includes/footer.md]: I added the trademark to the main page using a front matter variable in 5d113467a12.  Does this work?

Comment (ctubbsii) [inline _includes/footer.md]: Is this even required? It's messy to have it and is cleaner without, and I'm skeptical that it actually does anything to further protect the Apache brand (we already have "Apache" prefixes everwhere). So, if it's not required (and I can't find anything that says it is), I'd prefer it trimmed down.

**Comment by ctubbsii** [comment]
Spoke with Mike and made some suggestions. In summary:

Pros
* I like the improvements to the menu layout.
* I like the simplified footer.
* I like the image being moved into the menu.
* I like the removal of the search box. It wasn't very useful, I think.
* I like the removal of the left side social buttons on most pages.

Cons
* I don't think the index.md file needs to be renamed to index.html. It's still being processed as a Markdown file, because it has front-matter. Best to keep the filename consistency, I think, even if it currently doesn't contain markdown syntax. (Also easier to see the diff.)
* Menu doesn't resize quickly enough to the "mobile" view when the window shrinks. Might be best to shorten "Apache Software Foundation" or shrink the text size or something.
* "Apache License, Version 2.0" needs non-breaking spaces `&nbsp;` so it doesn't get broken up when window shrinks.
* I think the left-side social buttons shouldn't appear on the first page either. Instead, it should be moved to the footer. Or, at the very least, they should be made smaller without the large text next to them (hover text is sufficient).
* Nav bar getting bigger by X pixels means that the `h1::before`, etc. style needs to also increase by X pixels so section headers don't get hidden under the menu when navigating directly to a section.

File: _includes/footer.md (hunk 1 above)
Comment (joshelser) [inline _includes/footer.md]: > I added the trademark to the main page using a front matter variable in 5d11346. Does this work?

This looks great to me. I appreciate you being flexible.

> Is this even required?

tl;dr Given my current understanding, yes.

I think this is something that @billierinaldi and I had recently added to the website specifically to meet the branding requirements that the ASF sets forward. See http://www.apache.org/foundation/marks/pmcs.html#attributions:

> All project or product homepages must feature a prominent trademark attribution of all applicable Apache trademarks. Other project pages should feature attributions of any marks displayed thereon.

Policy aside, I believe reminding visitors of the website that all of these names (*full* names, even) are trademarks of the ASF and deserve proper attribution is a worthwhile goal. It really doesn't strike me as intrusive or distracting.

**Comment by mikewalch** [comment]
I addressed all of @ctubbsii cons except for moving the social buttons to the footer.  While I agree with his view, this pull request is getting big enough.  It should be fixed in a separate issue which I will create.

**Comment by mikewalch** [comment]
Created new issues for social buttons https://issues.apache.org/jira/browse/ACCUMULO-4316
